src/sage/rings/qqbar.py

```diff
@@ -3249,11 +3254,17 @@ def _interval_fast(self, prec):
         """
         return self._root._interval_fast(prec)
 
-    def union(self, other):
-        r""" Given generators ``alpha`` and ``beta``,
-        ``alpha.union(beta)`` gives a generator for the number field
+    def union(self, other, name='a'):
+        r"""
+        Given generators ``self``, `\alpha`, and ``other``, `\beta`,
+        ``self.union(other)`` gives a generator for the number field
         `\QQ[\alpha][\beta]`.
 
+        INPUT:
+
+        - ``other`` - an algebraic number
+        - ``name`` - string (default: ``'a'``); a name for the primitive element
```

Comment:
```suggestion
        - ``other`` -- an algebraic number
        - ``name`` -- string (default: ``'a'``); a name for the primitive element
```
there is also the second `-` needed to make the correct length hyphen.